Orange/tests/test_xlsx_reader.py

```diff
@@ -48,6 +48,11 @@ def test_named_sheet(self):
         self.assertEqual(len(table.domain.attributes), 4)
         self.assertEqual(table.name, 'header_0_sheet-my_sheet')
 
+    def test_named_sheet_table(self):
+        table = Table(get_dataset("header_0_sheet.xlsx"), sheet="my_sheet")
```

Comment:
True, that is exactly why I thought to include it here like this: that we would know what we break. :) But here I think it would actually be better to modify the constructor not to pass kwargs to from_file so that this would not even be allowed. What do you think?